api/idea-card/models/idea-card.settings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "kind": "collectionType",
+  "collectionName": "idea_cards",
+  "info": {
+    "name": "IdeaCard"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "ideaName": {
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "tagline": {
+      "type": "text",
+      "required": true
+    },
+    "description": {
+      "type": "text",
+      "required": true
+    },
+    "targetAudience": {
+      "type": "text",
+      "required": true
+    },
+    "timeCommitmentMin": {
```

Comment:
What unit is this?

Reply:
Do you mean what time unit like min or hours? I just set it to an integer, so the idea team can set a standard for what unit that should be.

Reply:
Yea I'm asking if it's minute or hour.

Reply:
I believe this will be hours. Strapi just asks if it’s gonna be an integer. It doesn’t let you specify the time unit.

Reply:
Maybe we can clarify that in the field name.